readAndExecute.go

```diff
@@ -287,6 +287,9 @@ func execEPN(result *answer, cmd *command, received *receivedStruct) {
 
 func fixReturnCodes(result *answer, config *configurationStruct, state *os.ProcessState) {
 	if result.returnCode >= 0 && result.returnCode <= 3 {
+		if config.workerNameInResult {
```

Comment:
why only add the worker name here and not for the other issues? If the plugin is not executable, this might be a useful information as well.
Also please apply "go fmt" or "make fmt".

Reply:
all other issues (that are out of bounds of plugin itself) already write worker name in the output. i have observed that the only scenario it does not write the worker name is when plugin is executed and returns "normal" answers. unless you mean something else ?

Reply:
that's exactly what i meant, but i haven't checked the code and couldn't remember anymore.